docs/zero.block

```diff
@@ -1,85 +1,62 @@
-                    "signature": "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
-                }
-            ],
-            "created_ts": 1501779527803139642,
+            "signatures": [],
+            "created_ts": 0,
             "creator_account_id": "",
             "tx_counter": 0,
             "commands": [
                 {
                     "command_type": "AddPeer",
-                    "address": "0.0.0.0:10001",
-                    "peer_key": "0000000000000000000000000000000000000000000000000000000000000000"
-                }
-            ]
-        },
-        {
-            "signatures": [
-                {
-                    "pubkey": "0000000000000000000000000000000000000000000000000000000000000000",
-                    "signature": "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
-                }
-            ],
-            "created_ts": 1501779527803139642,
-            "creator_account_id": "",
-            "tx_counter": 0,
-            "commands": [
+                    "address": "0.0.0.0:10000",
+                    "peer_key": "292a8714694095edce6be799398ed5d6244cd7be37eb813106b217d850d261f2"
+                },
                 {
                     "command_type": "CreateDomain",
                     "domain_name": "test"
                 },
                 {
-                    "command_type": "CreateAsset",
-                    "asset_name": "coin",
+                    "command_type": "CreateAccount",
+                    "account_name": "admin",
                     "domain_id": "test",
-                    "precision": 2
+                    "pubkey": "407e57f50ca48969b08ba948171bb2435e035d82cec417e18e4a38f5fb113f83"
                 },
                 {
                     "command_type": "CreateAccount",
+                    "account_name": "test",
                     "domain_id": "test",
-                    "account_name": "admin",
-                    "pubkey": "1000000000000000000000000000000000000000000000000000000000000000"
+                    "pubkey": "359f925e4eeecfdd6aa1abc0b79a6a121a5dd63bb612b603247ea4f8ad160156"
                 },
                 {
-                    "command_type": "CreateAccount",
+                    "command_type": "CreateAsset",
+                    "asset_name": "coin",
                     "domain_id": "test",
-                    "account_name": "test",
-                    "pubkey": "2000000000000000000000000000000000000000000000000000000000000000"
+                    "precision": 2
                 },
                 {
                     "command_type": "SetAccountPermissions",
                     "account_id": "admin@test",
                     "new_permissions": {
-                        "add_signatory": true,
+                        "add_signatory": false,
```

Comment:
May be it will be better to give admin account every possible permissions? I think it's acceptable for development environment.

Reply:
This will be changes to role system soon